roles/openshift_node/tasks/main.yml

```diff
@@ -59,6 +68,7 @@
     dest: /etc/sysconfig/{{ openshift.common.service_type }}-node
     regexp: "{{ item.regex }}"
     line: "{{ item.line }}"
+    create: yes
```

Comment:
Oh, I definitely think we need the task to modify the file, I just don't think we should be using the create argument.